terra-notebooks-playground/html_snapshots_of_notebooks.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
Can we have the naming convention of this file match the others? "HTML Snapshots Of Notebooks.ipynb"

Reply:
updated! Also added it to the README.